docs/advanced/input_files/input-main.md

```diff
@@ -2432,42 +2447,46 @@ These variables are used to control vdW-corrected related parameters.
 
 ## Berry phase and wannier90 interface
 
-These variables are used to control berry phase and wannier90 interface parameters.
+These variables are used to control berry phase and wannier90 interface parameters. [Detail introduce](https://github.com/deepmodeling/abacus-develop/blob/develop/docs/advanced/interface/Wannier90.md#wannier90)
 
 ### berry_phase
 
 - **Type**: Boolean
-- **Description**: 1, calculate berry phase; 0, not calculate berry phase.
-- **Default**: 0
+- **Description**: Controls the calculation of Berry phase
+  - true: Calculate Berry phase.
+  - false: Do not calculate Berry phase.
+- **Default**: false
 
 ### gdir
 
 - **Type**: Integer
-- **Description**:
-  - 1: calculate the polarization in the direction of the lattice vector a_1 that is defined in STRU file.
-  - 2: calculate the polarization in the direction of the lattice vector a_2 that is defined in STRU file.
-  - 3: calculate the polarization in the direction of the lattice vector a_3 that is defined in STRU file.
+- **Description**: Specifies the direction of the polarization in the lattice vector for Berry phase calculation
+  - 1: Calculate the polarization in the direction of the lattice vector a_1 defined in the STRU file.
+  - 2: Calculate the polarization in the direction of the lattice vector a_2 defined in the STRU file.
+  - 3: Calculate the polarization in the direction of the lattice vector a_3 defined in the STRU file.
 - **Default**: 3
 
 ### towannier90
 
 - **Type**: Integer
-- **Description**: 1, generate files for wannier90 code; 0, no generate.
+- **Description**: Controls the generation of files for the Wannier90 code
+  - 1: Generate files for the Wannier90 code.
+  - 0: Do not generate files for the Wannier90 code.
 - **Default**: 0
 
 ### nnkpfile
 
 - **Type**: String
-- **Description**: the file name when you run “wannier90 -pp ...”.
+- **Description**: The file name generated when running "wannier90 -pp ..." command.
 - **Default**: seedname.nnkp
 
 ### wannier_spin
 
 - **Type**: String
-- **Description**: If nspin is set to 2,
-  - up: calculate spin up for wannier function.
-  - down: calculate spin down for wannier function.
-- **Default**: up
+- **Description**: Specifies the spin direction for the Wannier function calculation when nspin is set to 2.
```

Comment:
- **Description**: the spin direction for the Wannier function calculation when nspin is set to 2